.changeset/shiny-pianos-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
+---
+
+ActionList.Item now supports link functionality through href and other anchor attributes, deprecating the separate ActionList.LinkItem component.
```

Comment:
Hi! 

What does the deprecation show up as? Does it show up as an error in CI for github-ui or only a soft suggestion in vscode?